packages/linter/README.md

```diff
@@ -56,11 +54,16 @@ performed.
 
 ## Development
 
-### Commands
+**Important note!** Many of the scripts below rely on binaries that are
+installed in the `../../node_modules/.bin` directory, and *will fail* if invoked
+in the default configuration. To fix this, either:
```

Comment:
this feels too complicated

Reply:
Yeah, it's not great. I could just remove these scripts, but it's useful to be able to run the linter tests individually. Is there a good way around this? I don't think any of the other packages support this? (i.e. running `npm test` from the cli when in the `packages/cache-url` directory.)

Reply:
Hm, I usually use the `fdescribe` feature to run specific specs while developing.